src/main/java/com/parksupark/soomjae/server/email/util/SecureCodeGenerator.java

```diff
@@ -0,0 +1,19 @@
+package com.parksupark.soomjae.server.email.util;
+
+import java.security.SecureRandom;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SecureCodeGenerator {
+
+    private final SecureRandom secureRandom = new SecureRandom();
+    private static final String SAFE_CHARS = "23456789ABCDEFGHJKMNPQRSTUVWXYZ";
+
+    public String generateVerificationCode() {
+        StringBuilder code = new StringBuilder();
```

Comment:
p5;  

```suggestion
        StringBuilder code = new StringBuilder(6);
```

위와 같이 초기 용량을 정해줄 수 있습니다. magic number 6도 처리해주는 것이 좋을 것 같습니다. ex) CODE_LENGTH